study.md

```diff
@@ -0,0 +1,53 @@
+# Ember Object Study
+
+Use your favorite search engine and the provided readings to research and
+respond to the following questions.
+
+In your responses, be sure to cite any relevant sources you consulted in your
+search. We ask you to write responses in your own words in order to see how you
+process what you've read. Please do not respond with direct quotes from source
+material. Instead, digest what you've read and repeat it in your own voice.
+
+## Required Readings
+
+-   [The Object Model](https://guides.emberjs.com/v2.11.0/object-model/) (The sections marked as (optional) are only if you're interested in reading further)
```

Comment:
This sentence is weird, reflow? maybe `The sections marked as (optional) are only for if you're interested in reading further` Something like that. Non-critical can be fixed or mad an issue for next iteration.